gems/sorbet-runtime/tools/generate_call_validation.cc

```diff
@@ -121,17 +121,7 @@ void generateCreateValidatorFast(const Options &options, ValidatorKind kind, siz
     }
     fmt::print("&blk|\n");
 
-    fmt::print("      # This block is called for every `sig`. It's critical to keep it fast and\n"
-               "      # reduce number of allocations that happen here.\n"
```

Comment:
ooc why have these comments been dropped?

Reply:
It's not literally true - none of these individual methods is called for every `sig` anymore - and it didn't seem like it was adding enough value to be worth trying to reword.

Reply:
wfm